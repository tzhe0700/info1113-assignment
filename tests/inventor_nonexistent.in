ADD COLLECTION spellbooks.csv
INVENTOR Nonexistent Author
EXIT
