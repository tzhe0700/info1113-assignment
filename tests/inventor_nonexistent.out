17 spellbooks successfully added.
No spellbooks by Nonexistent Author.
